### PR TITLE
[PaloAltoNetworks] Fix the segments and the resource provider to be with the right casing

### LINKED ISSUE #2625: PaloAltoNetworks: fix resource ID segments
Hi @stephybun , could you fix this issue asap since service team expects this RP to be released this month? Thanks.

The related links:
https://github.com/hashicorp/pandora/pull/2617
https://github.com/hashicorp/pandora/pull/2618

### PR DESCRIPTION
**Pull request by neil-yechenwei**
Fixes #2625

### CODE REVIEW

File: tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go [hunk 1]
```diff
@@ -359,6 +363,7 @@ func NormalizeResourceProviderName(input string) string {
 		"microsoft.windowsesu":                     "Microsoft.WindowsESU",
 		"microsoft.windowsiot":                     "Microsoft.WindowsIoT",
 		"microsoft.workloadmonitor":                "Microsoft.WorkloadMonitor",
+		"PaloAltoNetworks.Cloudngfw":               "PaloAltoNetworks.CloudNGFW",
```
Comment (tombuildsstuff) [inline tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go:366]: The Key here needs to be lower-cased (see the usage below), so this wants to be:

```suggestion
		"paloaltonetworks.cloudngfw":               "PaloAltoNetworks.CloudNGFW",
```

Comment (neil-yechenwei) [reply tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go]: Thanks. Updated.

File: tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go [hunk 2]
```diff
@@ -135,6 +137,7 @@ func NormalizeSegment(input string, camelCase bool) string {
 		"ipconfigurations":                        "ipConfigurations",
 		"iscsiservers":                            "iscsiServers",
 		"linkedservices":                          "linkedServices",
+		"localRulestacks":                         "localRuleStacks",
```
Comment (tombuildsstuff) [inline tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go:140]:
```suggestion
		"localrulestacks":                         "localRuleStacks",
```

File: tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go [hunk 3]
```diff
@@ -122,9 +122,11 @@ func NormalizeSegment(input string, camelCase bool) string {
 		"fhirservices":                      "fhirServices",
 		"fileservers":                       "fileServers",
 		"fallbackroute":                     "fallbackRoute",
+		"fqdnlists":                         "fqdnLists",
 		"fqn":                               "fqn", // handles FQN
 		"frontdoorwebapplicationfirewallpolicies": "frontDoorWebApplicationFirewallPolicies",
 		"functionappsettings":                     "functionAppSettings",
+		"globalRulestacks":                        "globalRuleStacks",
```
Comment (tombuildsstuff) [inline tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go:129]:
```suggestion
		"globalrulestacks":                        "globalRuleStacks",
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
These also need to be lower-cased

File: tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go (hunk 3 above)
Comment (neil-yechenwei) [reply tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go]: Updated

File: tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go (hunk 2 above)
Comment (neil-yechenwei) [reply tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go]: Updated

Comment (neil-yechenwei) [reply tools/importer-rest-api-specs/components/parser/cleanup/cleanup.go]: Updated

**Review by stephybun — APPROVED** [review]
Thanks @neil-yechenwei LGTM 👍

**Pull request merged**